docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
 
   postgres:
     restart: always
-    image: postgres:11.4-alpine
+    image: postgres:11-alpine
```

Comment:
Any reason why using 11 and not 11.4 ? I don't know anything about those, just asking out of curiousity 😊.

Reply:
I guess 11 will auto update to 11.5, 11.6 and so on.